wercker build panics when build pipeline is empty

Given a `wercker.yml` file with the following contents:
```yaml
box: golang:1.6.2
build:
```

running `wercker build` panics. This is because on [line 37 of docker/pipeline.go](https://github.com/wercker/wercker/blob/c1629b0ed455c07aaca93378d2ab34675c8fefa9/docker/pipeline.go#L37), `pipelineConfig` is `nil`, and `ok` is still `true`, so when it gets down to [line 43](https://github.com/wercker/wercker/blob/c1629b0ed455c07aaca93378d2ab34675c8fefa9/docker/pipeline.go#L43) it freaks out. A simple if statement checking for a `nil` value would probably be enough to fix this bug.

I'll see if I have time this weekend to write a test and make PR.

```
$ wercker build
--> No Docker host specified, checking: unix:///var/run/docker.sock
--> Executing pipeline
--> Running step: setup environment
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x5d28d7]

goroutine 1 [running]:
github.com/wercker/wercker/docker.NewDockerPipeline(0xadd718, 0x5, 0xc820096d20, 0xc820180000, 0xc820128c30, 0x7fa3231c9be8, 0xc820071c30, 0x7fa3231c9be8, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/docker/pipeline.go:43 +0x207
github.com/wercker/wercker/docker.NewDockerBuild(0xadd718, 0x5, 0xc820096d20, 0xc820180000, 0xc820128c30, 0x7fa3231c9be8, 0xc820071c30, 0x10, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/docker/build.go:29 +0x85
github.com/wercker/wercker/cmd.GetBuildPipelineFactory.func1(0xc820096d20, 0xc820180000, 0xc820128c30, 0x0, 0x0, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/cmd/runner.go:59 +0x2f1
github.com/wercker/wercker/cmd.(*Runner).GetPipeline(0xc8200160a0, 0xc820096d20, 0x0, 0x0, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/cmd/runner.go:353 +0x53
github.com/wercker/wercker/cmd.(*Runner).SetupEnvironment(0xc8200160a0, 0x7fa323289140, 0xc8200122d0, 0x0, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/cmd/runner.go:460 +0x9b2
github.com/wercker/wercker/cmd.executePipeline(0x7fa323289140, 0xc8200122d0, 0xc820180000, 0xc820128c30, 0xc82000a760, 0x0, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/cmd/main.go:910 +0xd08
github.com/wercker/wercker/cmd.cmdBuild(0x7fa323289140, 0xc8200122d0, 0xc820180000, 0xc820128c30, 0x0, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/cmd/main.go:413 +0x37b
github.com/wercker/wercker/cmd.glob.func1(0xc8200d4540)
        /gopath/src/github.com/wercker/wercker/cmd/main.go:69 +0x7f3
github.com/wercker/wercker/vendor/github.com/codegangsta/cli.Command.Run(0xadd718, 0x5, 0xadd550, 0x1, 0xaed460, 0xf, 0x0, 0x0, 0x0, 0x0, ...)
        /gopath/src/github.com/wercker/wercker/vendor/github.com/codegangsta/cli/command.go:113 +0x10ab
github.com/wercker/wercker/vendor/github.com/codegangsta/cli.(*App).Run(0xc8201260d0, 0xc820082000, 0x2, 0x2, 0x0, 0x0)
        /gopath/src/github.com/wercker/wercker/vendor/github.com/codegangsta/cli/app.go:156 +0xbda
main.main()
        /gopath/src/github.com/wercker/wercker/main.go:25 +0x49

goroutine 18 [syscall]:
os/signal.loop()
        /usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
        /usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 19 [select, locked to thread]:
runtime.gopark(0xc0b740, 0xc820025728, 0xadfea0, 0x6, 0x42bd18, 0x2)
        /usr/local/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc820025728, 0x0, 0x18)
        /usr/local/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc820025728)
        /usr/local/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
        /usr/local/go/src/runtime/signal1_unix.go:227 +0x353
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 20 [chan receive]:
github.com/wercker/wercker/util.(*SignalMonkey).Register.func1(0xc820078cc0)
        /gopath/src/github.com/wercker/wercker/util/signal.go:95 +0x76
created by github.com/wercker/wercker/util.(*SignalMonkey).Register
        /gopath/src/github.com/wercker/wercker/util/signal.go:106 +0x111

goroutine 21 [chan receive]:
github.com/wercker/wercker/util.(*SignalMonkey).Register.func1(0xc820078d00)
        /gopath/src/github.com/wercker/wercker/util/signal.go:95 +0x76
created by github.com/wercker/wercker/util.(*SignalMonkey).Register
        /gopath/src/github.com/wercker/wercker/util/signal.go:106 +0x111

```

--- a/docker/pipeline.go
+++ b/docker/pipeline.go
@@ -39,6 +39,10 @@ func NewDockerPipeline(name string, config *core.Config, options *core.PipelineO
 		return nil, fmt.Errorf("No pipeline named %s", pipelineName)
 	}
 
+	if pipelineConfig == nil {
+		return nil, fmt.Errorf("Pipeline %s is empty", pipelineName)
+	}
+
 	// Select this pipeline's config or the global config
 	rawBoxConfig := pipelineConfig.Box
 	if rawBoxConfig == nil {

